html-forms/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get" class="html_forms">
+
+      <div class="html_forms">
+        <label for="email">Email</label>
+      </div>
+
+      <div class="html_forms">
+        <input type="email" id="mail" name="user_email"/>
+      </div>
+
+      <div class="html_forms">
+        <label for="question">Why is coding cool?</label>
+      </div>
+
+      <div class="html_forms">
+        <select id="reason" name="user_opinion">
+          <option>Select an option</option>
+          <option>Because it's interesting</option>
+          <option>Because it's cool</option>
+          <option>Because it's powerful</option>
```

Comment:
make sure these `option` tags have a `value` attribute, this is what will get sent in the form submission.